Source/Plugin.BLE.Abstractions/Utils/BleCommandQueue.cs

```diff
@@ -99,7 +99,7 @@ public async Task ExecuteAsync()
             {
                 IsExecuting = true;
                 var source = _taskSource();
-                if (source != await Task.WhenAny(source, Task.Delay(TimeoutInMiliSeconds)))
+                if (source != await Task.WhenAny(source, Task.Delay(TimeoutInMiliseconds)))
```

Comment:
Another typo here, as detected by the [GH Actions build](https://github.com/xabre/xamarin-bluetooth-le/pull/533/checks?check_run_id=3393160046):

```
BleCommandQueue.cs(102,69): error CS0103: The name 'TimeoutInMiliseconds' does not exist in the current context
```

Have you ever even tried to compile this branch?!?